homeassistant/components/ecovacs/strings.json

```
@@ -113,6 +119,12 @@
       "lifespan_side_brush": {
         "name": "Side brushes lifespan"
       },
+      "lifespan_unit_care": {
+        "name": "Unit Care lifespan"
```

```suggestion
        "name": "Unit care lifespan"
```

What is the unit care?